src/components/UserInfo/UserInfo.jsx

```diff
@@ -1,4 +1,10 @@
-// Add the required props
-export const UserInfo = () => (
-  <>UserInfo markup</>
+import './UserInfo.scss';
+
+export const UserInfo = ({ user: { email, name } }) => (
+  <a
```

Comment:
Would be better to destructure the user prop inside the component to make code more readable